chef_master/source/install_server_pre.rst

```diff
@@ -78,47 +78,40 @@ Hardware Requirements
 =====================================================
 .. tag system_requirements_server_hardware
 
-All machines in a Chef server deployment (including a standalone Chef Analytics machine) have the following hardware requirements.
+All machines in a Chef server deployment have the following hardware requirements. Disk space for standalone and backend servers should scale up with the number of nodes that the servers are managing. A good rule to follow is to allocate 2 MB per node. The disk values listed below should be a good default value that you will want to modify later if/when your node count grows.
 
 For all deployments:
 
 * 64-bit architecture
 
 For a standalone deployment:
 
-* 4 total cores, 2.0 GHz AMD 41xx/61xx or Intel Xeon 5000/E5 CPUs
-* 4 GB of RAM; 8 GB of RAM for Chef Analytics
+* 4 total cores (physical or virtual)
+* 8 GB of RAM or more
 * 5 GB of free disk space in ``/opt``
 * 5 GB of free disk space in ``/var``
 
-For a tiered deployment:
+For a high availability deployment:
 
-* 8 total cores 2.0 GHz AMD 41xx/61xx or Intel Xeon 5000/E5 CPUs or faster
-* 16GB RAM
-* 2 x 300GB SAS RAID1 drives
-* Hardware RAID card
-* 1 GigE NIC interface
-* 20 GB of free disk space in ``/opt``
-* 40 GB of free disk space in ``/var``
-* A back-end server; all other systems will be front-end servers.
+* Three backend servers; as many frontend servers as required
 
-For a high availability deployment:
+.. tag system_requirements_ha
```

Comment:
This might be easier to read if it were broken up in FE and BE sections.  
Backend
  - Backend requirements

Frontend
  - Frontend requirements